isis/src/rosetta/apps/rososiris2isis/PvlTranslationTable.cpp

```diff
@@ -0,0 +1 @@
+/scratch/ISIS3/isis/src/base/objs/PvlTranslationTable/PvlTranslationTable.cpp
```

Comment:
I'm not sure what this is, but I think it should be removed from the PR. Is this a symlink to `/scratch/ISIS3/isis/src/base/objs/PvlTranslationTable/PvlTranslationTable.cpp` from your `isis/src/rosetta/apps/rososiris2isis` area? I am not sure how this shows up in a PR.